src/Command/GetGoMRIStatisticsCommand.php

```diff
@@ -98,6 +112,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $io->writeln("Total number of GoMRI datasets submitted since 2021-01-01 until current date " . $totalGomriDatasetSubmittedSince2021Count);
         $io->writeln("Total number of datasets (all data including GoMRI) submitted since 2021-01-01 until current date " . $totalDatasetSubmittedSince2021Count);
 
+        if ($start != '' and $end != '') {
+            $io->writeln("Total GoMRI Downloads from $start to $end: " . $this->logActionItemRepository->countDownloads($start, $end));
```

Comment:
Turn into a proper tz containing datetime object and pass those instead of strings.